### PR TITLE
add layout ,styles ,table to letters page and create new comp inside …

### PR DESCRIPTION
**Pull request by Amrillo**
Added layout with table and styles to letters page as well as created -create-letters-template - component inside letters page

### CODE REVIEW

File: src/app/app.module.ts [hunk 1]
```diff
@@ -30,6 +29,9 @@ import { MatDialogModule } from '@angular/material/dialog';
 import { LoginComponent } from './login/login.component';
 import {MatCardModule} from '@angular/material/card';
 import {MatToolbarModule} from '@angular/material/toolbar';
+import { CreateLettersTempComponent } from './components/letters/create-letters-temp/create-letters-temp.component';
+import {MatTableModule} from '@angular/material/table';
```
Comment (AudriusBaltusis) [inline src/app/app.module.ts:33]: Please follow coding style, there should be spaces withing import braces

File: src/app/components/letters/create-letters-temp/create-letters-temp.component.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+@Component({
+  selector: 'app-create-letters-temp',
+  templateUrl: './create-letters-temp.component.html',
+  styleUrls: ['./create-letters-temp.component.scss']
+})
+export class CreateLettersTempComponent implements OnInit {
+
+  constructor() { }
+
+  ngOnInit(): void {
```
Comment (AudriusBaltusis) [inline src/app/components/letters/create-letters-temp/create-letters-temp.component.ts:12]: Please remove empty `ngOnInit`

File: src/app/components/letters/letters.component.ts [hunk 3]
```diff
@@ -7,9 +20,13 @@ import { Component, OnInit } from '@angular/core';
...
 
-  constructor() { }
+  displayedColumns: string[] = ["dateUpdate", "Name","edit"];
+  dataSource = new MatTableDataSource(table_Data)
 
-  ngOnInit(): void {
+  constructor() {
   }
 
+  ngOnInit(): void {
```
Comment (AudriusBaltusis) [inline src/app/components/letters/letters.component.ts:29]: Please remove empty `ngOnInit`

**Review by AudriusBaltusis — APPROVED** [review]

File: src/app/app.module.ts (hunk 1 above)
Comment (Amrillo) [reply src/app/app.module.ts]: I corrected and pushed commits.

File: src/app/components/letters/create-letters-temp/create-letters-temp.component.ts (hunk 2 above)
Comment (Amrillo) [reply src/app/components/letters/create-letters-temp/create-letters-temp.component.ts]: Done!!!

File: src/app/components/letters/letters.component.ts (hunk 3 above)
Comment (Amrillo) [reply src/app/components/letters/letters.component.ts]: Done !!!

**Pull request merged**